src/features/broker-protection/click.js

```diff
@@ -0,0 +1,19 @@
+import { getElement } from './utils.js'
+import { ErrorResponse, SuccessResponse } from './types.js'
+
+/**
+ * @param action // TODO: get type based on actionType
+ * @return {import('./types.js').ActionResponse}
+ */
+export function click (action) {
```

Comment:
heh, this is so similar to what we have in [autoconsent](https://github.com/duckduckgo/autoconsent/blob/main/lib/rule-executors.ts#L142). Just a note